clients/src/main/java/org/apache/kafka/common/protocol/SendBuilder.java

```diff
@@ -39,6 +42,7 @@
  */
 public class SendBuilder implements Writable {
     private final Queue<Send> sends = new ArrayDeque<>();
+    private final List<ByteBuffer> buffers = new ArrayList<>();
```

Comment:
why ```sends``` is ```ArrayDeque``` and ```buffers``` is ```ArrayList```?

Reply:
Oh, just because `MultiRecordsSend` needs a Queue.